packages/client/webapp/src/utils/TA2/stratify.ts

```diff
@@ -0,0 +1,25 @@
+// Providing the ID of 3 Models (model A, model B, and the type Model)
+// Create a new model of based off of the stratification
+
+import { json } from 'd3';
+
+// TODO: Better error handling (models not found for example)
+//       Override function to also accept 2 typed models (need to chat with TA2 to fully understand)
+export async function stratify(modelA, modelB, typeModel) {
```

Comment:
Nitpick: this might be more accurately titled `fetchStratificationResult(...)` which would serve the double purpose of making it easier to avoid name clashes when importing later, e.g.
```
import { fetchStratificationResult } from '@/utils/TA2/stratify';
fetchStratificationResult()
// instead of
import * as strat from '@/utils/TA2/stratify';
strat.stratify()
```

Reply:
Good point - done